helpdesk_timesheet/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Georg Notter <georg.notter@agenterp.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models
```

Comment:
You can group this lines
```suggestion
from odoo import models, fields, api, _
```

Reply:
done